IDEAS/Buildings/Components/Window.mo

```diff
@@ -271,8 +269,9 @@ equation
   connect(shaType.hForcedConExt, radSolData.hForcedConExt) annotation (
     Line(points={{-68.5,-32.7043},{-76,-32.7043},{-76,-62.2},{-79.4,-62.2}},
                                                                     color = {0, 0, 127}));
-  connect(outsideAir.TDryBul_in, shaType.TDryBul) annotation(
-    Line(points = {{-42, -90}, {-46, -90}, {-46, -48}, {-58, -48}}, color = {0, 0, 127}));
+  connect(outsideAir.TDryBul_in, shaType.TDryBul) annotation (
+    Line(points={{-42,-90},{-46,-90},{-46,-49.4895},{-57.5,-49.4895}},
+                                                                    color = {0, 0, 127}));
```

Comment:
This is a GUI change which seems unintended. Could you change it back to the original one?
If this doesn't work in Dymola, you can use a text editor as well to only change the desired code.